containers/images/content-server/Dockerfile

```diff
@@ -13,9 +13,11 @@ RUN ln -s /etc/pki/pulp/certs/pulp.crt /etc/pki/tls/certs/localhost.crt
 RUN ln -s /etc/pki/pulp/private/pulp.key /etc/pki/tls/private/localhost.key
 
 RUN chown apache:apache /var/lib/pulp
+RUN mkdir /var/www/html/pub && chown apache:apache -R /var/www/html
 
 RUN sed -i 's/127.0.0.1/squid/g' /etc/httpd/conf.d/pulp_streamer.conf
 
 COPY container-assets/content_wsgi.conf /etc/httpd/conf.d/content_wsgi.conf
 
 CMD ["httpd", "-D", "FOREGROUND"]
+ENTRYPOINT []
```

Comment:
Can you move `ENTRYPOINT` above `CMD`?

I think it reads better as technically `CMD` is supposed to be the arguments for `ENTRYPOINT`

Reply:
Also, is there any way to make `ENTRYPOINT` more generic in the pulp image rather than having to blank it here?

Reply:
The tough part is Pulp's entrypoint is doing things like database operations before handing off to the final run command. I could move all that into the CMD itself but I had been following most examples I found that put things like database operations into the entrypoint.